src/Blocks/BlockButton.h

```diff
@@ -128,6 +128,13 @@ class cBlockButtonHandler :
 		UNUSED(a_Meta);
 		return 0;
 	}
+
+
+	static const bool IsButtonOn(NIBBLETYPE a_BlockMeta)
```

Comment:
did you mean `static bool IsButtonOn(...) const` because currently th const is applied to the bool, not the this pointer.